### PR TITLE
added components async and added vuex

### PR DESCRIPTION
**Pull request by Wellington01**
#56

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -396,75 +150,42 @@ import {
...
     }),
 
     computed: {
-        tablesFiltered() {
+        tablesFiltered () {
             return this.tables.filter(table => {
                 return (
                     !this.tableFilter || table.indexOf(this.tableFilter) !== -1
                 )
             })
```
Comment (gabrielboliveira) [inline src/App.vue]:
```js
return this.tables.filter(table => !this.tableFilter || table.indexOf(this.tableFilter) !== -1)
```

File: src/App.vue [hunk 2]
```diff
@@ -396,75 +150,42 @@ import {
     mapKeys as _mapKeys,
     omit as _omit,
     without as _without,
+    defer as _defer,
 } from 'lodash'
```
Comment (gabrielboliveira) [inline src/App.vue]: double check if you are using all these imports

File: src/app/component/explorer/query.vue [hunk 3]
```diff
@@ -0,0 +1,78 @@
+<template>
+    <div id="query">
+        <el-form>
+            <el-form-item>
+                <el-input type="textarea" :autosize="{ minRows: 10 }" placeholder="Execute Query" v-model="query"></el-input>
```
Comment (gabrielboliveira) [inline src/app/component/explorer/query.vue]: break into new lines

File: src/app/global/error.vue [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<template>
+    <div class="error">
+       <i class="el-icon-circle-close"></i> Could not load component.
```
Comment (gabrielboliveira) [inline src/app/global/error.vue]: tabulation

**Review by gabrielboliveira — CHANGES_REQUESTED** [review]

File: src/app/component/explorer/structure.vue [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+                        <div class="cell" v-text="row.column_comment"></div>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</template>
+
+<script>
+import ConnectionMixin from '../../../js/mixin/connection.js'
```
Comment (dbpolito) [inline src/app/component/explorer/structure.vue]: i think we have `~` alias, if not add it

**Comment by Wellington01** [comment]
changed

**Pull request merged**